src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -207,13 +222,14 @@ public void simulationPeriodic() {
         simDriveMotor.update(Robot.kDefaultPeriod);
         simTurningMotor.update(Robot.kDefaultPeriod);
 
-        simDriveEncoder.setPosition(simDriveEncoder.getPosition() + simDriveMotor.getAngularVelocityRadPerSec() * Robot.kDefaultPeriod);
-        simDriveEncoder.setSimVelocity(simDriveMotor.getAngularVelocityRadPerSec());
+        int errorCode = simDriveEncoder.setPosition(simDriveEncoder.getPosition() + simDriveMotor.getAngularVelocityRadPerSec() * Robot.kDefaultPeriod).value;
```

Comment:
What are the error codes for simDriveEncoder's?  How will they fail?  
Add a comment, I am not sure this is useful

Reply:
leaving this for next review, I don't know enough about the subsystem and kennyson isn't here for me to ask

Reply:
actually seems the issue has already been addressed